src/main/java/org/swisspush/redisques/QueueStatsService.java

```diff
@@ -113,7 +117,7 @@ public <CTX> void getQueueStats(CTX mCtx, GetQueueStatsMentor<CTX> mentor) {
         }
     }
 
-    private <CTX> void fetchQueueNamesAndSize(GetQueueStatsRequest<CTX> req, BiConsumer<Throwable, GetQueueStatsRequest<CTX>> onDone) {
+    <CTX> void fetchQueueNamesAndSize(GetQueueStatsRequest<CTX> req, BiConsumer<Throwable, GetQueueStatsRequest<CTX>> onDone) {
```

Comment:
Do not change visibility of methods/variables just to test them. This is bad practice. If you can find a workaround or there is a arguable reason, it may be ok.

Reply:
Ok.